installation/release-notes-main/release-notes-9-0-0.adoc

```diff
@@ -0,0 +1,77 @@
+= BAMOE, 9.0.0, release notes
+include::../../partials/attributes.adoc[]
+
+Highlights, new features, removed components, and differences between {PRODUCT} {VERSION_OLD} and {VERSION}:
+
+== Components
+
+* Runtimes
+** Decision Service
+** Observability
+*** Auditability (Metrics)
+*** Tracing via Kafka
+
+== New Components
+
+* IBM Business Automation Manager Open Editions Canvas 
+** Accelerators
+*** Work with individual assets or full projects
+*** Simple projects visualization
+*** Easy branch management
+*** Share
+**** download full projects
+**** individual decision files (DMN)
+**** svgs
+**** or obtain an embeddable version of your decision
+** Improved Collaboration
+*** Support for GitHub, GitHub Enterprise, Bitbucket Cloud
+*** Seamless git integration
+**** connect 
+**** clone (public and private repositories) 
+**** pull 
+**** push
+**** commit with custom messages
+** Decision Test capabilities
+*** Automatically generated forms 
+*** Export for 
+** One-click deploy (Dev/test purposes)
+*** OpenShift environments
+*** Kubernetes
+** Run on containerized environments with new product image 
+* IBM Business Automation Manager Open Editions Developer Tools
+** Available on the marketplace
+* Runtime support for DMN 1.4
+* Accelerators
+* Upstream-first 
+** Upstream certified versions available through new IBM Bills Of Material (dependency management)
+* Cloud
+** Helm Charts
```

Comment:
Helm is not included

Reply:
This slipped through the cracks in 9.0.0. We said it was included in 9.0.0 :)

Reply:
Removed